py/desispec/quicklook/__init__.py

```diff
@@ -6,7 +6,7 @@
 from __future__ import absolute_import, division, print_function
 
 #SE: needed to maintain quicklook-specific version 
-__qlversion__ = "10.18.0"
+__qlversion__ = "02.19.0"
```

Comment:
Suggestion: reorder your QL version numbering schedule to be YY.MM.minor so that they will be sorted in time.  02.19.0 implies that this version is earlier than 10.18.0 until a human interprets it as a date.  i.e. let's switch this version to 19.2.0